onnxruntime/test/debug_node_inputs_outputs/debug_node_inputs_outputs_utils_test.cc

```diff
@@ -29,7 +29,7 @@ void VerifyTensorProtoFileData(const PathString& tensor_proto_path, gsl::span<co
   actual_data.resize(expected_data.size());
   ASSERT_STATUS_OK(utils::UnpackTensor(tensor_proto, Path{}, actual_data.data(), actual_data.size()));
 
-  ASSERT_EQ(gsl::make_span(actual_data), expected_data);
+  ASSERT_EQ(gsl::make_span(static_cast<gsl::span<const T>>(actual_data)), expected_data);
```

Comment:
maybe just use the span ctor directly
```suggestion
  ASSERT_EQ(gsl::span<const T>(actual_data), expected_data);
```

Reply:
Thanks - I just thought the same thing :). Just issued an update a little while ago.